products/appengine/terraform.yaml

```
@@ -87,6 +87,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         update_mask_fields:
           - "ssl_settings.certificate_id"
           - "ssl_settings.ssl_management_type"
+  ApplicationUrlDispatchRules: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: "{{project}}"
+    import_format: ["{{project}}"]
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "app_engine_application_url_dispatch_rules_basic_example"
```

```suggestion
        name: "app_engine_application_url_dispatch_rules_basic"
```

Oops- the example suffix is added elsewhere. Do you mind amending this + the filename? Sorry about the backtracking!

---

done